lib/Doctrine/ODM/PHPCR/Query/Builder/QueryBuilder.php

```diff
@@ -80,7 +80,7 @@ public function getQuery()
     /**
      * @param BuilderConverterPhpcr $converter
      */
-    public function setConverter(BuilderConverterPhpcr $converter)
+    public function setConverter(BuilderConverterInterface $converter)
```

Comment:
was going to say this is a BC break but actually not, as BuilderConverterPhpcr was more restricted. that class implements the interface so people extending BuilderConverterPhpcr would still be able to call this method with their extending class.